src/StmtToHtml.cpp

```diff
@@ -1000,7 +1000,7 @@ class StmtToHtml : public IRVisitor {
     }
 
     StmtToHtml(const string &filename)
-        : id_count(0), context_stack(1, 0) {
+        : context_stack(1, 0) {
```

Comment:
Here's another case where it doesn't depend on the args - it's just not the sort of constant that's blessed with member initialization